### ISSUE TITLE
Unable to save file in Linux Mint xfce

### ISSUE THREAD

**Issue by tspantu** [chiragsakhuja/lc3v2#37]
I recently put linux mint on one of my old laptops to give some life to it and one of the first things i did was try and install lc3 since i use it all the time at school. I downloaded the linux version then changed it to an exe file in the terminal as i saw in the install instructions, but there it said that after opening, a window would appear that would start the installation process. No such window has appeared and I've been able to use the lc3 as an exe file, but i'm unable to save any files or assemble them. Whenever i try to save anything, the usual popup appears and asks you to select a destination folder, i can navigate through the folders just fine, but the save button just doesn't work when I press it. I've tried uninstalling and the installing again, hoping that the window with the installation might appear but no luck. Does anybody have any idea if this might be an issue with lc3 itsself or if it might be a problem with linux mint (i'm using the latest version of xfce).
Thank you in advance!

**Comment by chiragsakhuja** [comment]
Would you mind sharing the solution? Since Linux is my least tested platform for the GUI, I was Just wondering what you may have been running into.

**Comment by tspantu** [comment]
Well, first I tried installing it again but nothing was different, so I checked that the glibc version was the newest one, installed an app opener that read the exe files better, but still no result. In the end, after leaving the comment and tried a million other things in the terminal, I realised that i was dumb and I wasn't naming the file and so it couldn't be saved... Happily, there was no problem after all! Also, thank you for work! I really quite like the program and so does my university!
